c2corg_api/search/sync.py

```diff
@@ -62,13 +64,27 @@ def remove_doc():
             # https://github.com/c2corg/v6_api/issues/89
             DBSession.refresh(document)
 
+        is_user_profile = isinstance(document, UserProfile)
+        if is_user_profile:
+            # set user login + full-name as document title so that it can
+            # be searched
+            user_login, user_name = DBSession. \
+                query(User.name, User.username). \
+                filter(User.id == document.document_id). \
+                first()
+            user_title = '{0} {1}'.format(user_name or '', user_login or '')
```

Comment:
Makes sense. @gberaudo do you know what are the relevant usernames you have discussed with the client. It seems that some fields must be kept secret (login?)? Perhaps we should only index public usernames such as topo_name and forum_name (I don't remember the names).